eue/src/main/java/ch/cyberduck/core/eue/EueMultipartWriteFeature.java

```diff
@@ -96,8 +96,8 @@ public HttpResponseOutputStream<EueWriteFeature.Chunk> write(final Path file, fi
             throw new ChecksumException(LocaleFactory.localizedString("Checksum failure", "Error"), e);
         }
         return new HttpResponseOutputStream<EueWriteFeature.Chunk>(new MemorySegementingOutputStream(proxy,
-                new HostPreferences(session.getHost()).getInteger("eue.upload.multipart.size")),
-                new EueAttributesAdapter(), status) {
+            new HostPreferences(session.getHost()).getInteger("eue.upload.multipart.size")),
```

Comment:
this constant reindenting is super annoying, can we get some unified code styles enforced?

Reply:
Agree. We have `.idea/codeStyleSettings.xml` which should ensure a unified style. Behaviour changes from version to version. Should probably open an issue with the JetBrains guys.

Reply:
Maybe there are settings in codeStyleSettings that are deprecated/replaced/removed, and user-defaults are taken, which differ from those originally set in codeStyleSettings?